docs/projects/modules/gsoc/pages/2022/project-ideas/plugin-installation-manager-tool.adoc

```diff
@@ -3,8 +3,8 @@ layout: gsocprojectidea
 title: "Plugin Installation Manager Tool Improvements"
 goal: "Introduce new features and improvements in the plugin installation manager"
 category: Tools
-year: 2023
-status: published
+year: 2024
+status: draft
```

Comment:
The year as suggested by the file name should be 2022 and neither 2023 or 2024. Could you please check again?